Extension/c_cpp_properties.schema.json

```diff
@@ -64,7 +64,7 @@
                         "type": "string"
                     },
                     "includePath": {
-                        "description": "A list of paths for the IntelliSense engine to use while searching for included headers. Searching on these paths is not recursive.",
+                        "description": "A list of paths for the IntelliSense engine to use while searching for included headers. Searching on these paths is not recursive. Specify '**' to indicate recursive search. For example, '${workspaceFolder}/**' will search through all subdirectories while '${workspaceFolder}' will not.",
```

Comment:
I forgot about the html of the configuration UI will also need an update. See file https://github.com/microsoft/vscode-cpptools/blob/master/Extension/ui/settings.html

Reply:
The description in the settings.html seems good to me.